bw2io/backup.py

```diff
@@ -61,25 +71,38 @@ def backup_project_directory(project: str):
     if project not in projects:
         raise ValueError("Project {} does not exist".format(project))
 
+    dir_backup = dir_backup or os.path.expanduser("~")
+
+    if timestamp:
```

Comment:
Not a huge fan of overwriting this variable name.

Reply:
I can change it to
`if not dir_backup:  dir_backup = os.path.expanduser("~")` 
but isn't that essentially the same logic written differently?

Reply:
Or do you mean have it as a separate variable like:

`timestamp_str = f'.{datetime.datetime.now().strftime("%d-%B-%Y-%I-%M%p")}' if timestamp else ""`
`backup_filename = f"brightway2-project-{project}-backup{timestamp_str}.tar.gz"`